.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Suggest a feature or improvement
+title: "[FEATURE]: "
+labels: kind/enhancement
+assignees: ''
+
+---
+
+<!-- MAKE SURE TO APPLY SUITABLE LABELS BEFORE FILING THE ISSUE, ESPECIALLY PRIORITY -->
+<!-- AND `kind/documentation` IF IT INVOLVES DOCUMENTATION IN ANY WAY e.g. README -->
+## Details
+
+**Is your feature request related to a problem? Please describe.**
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+
+**Describe the solution you'd like**
+<!-- A clear and concise description of what you want to happen. -->
+
+**Describe alternatives you've considered**
+<!-- A clear and concise description of any alternative solutions or features you've considered. -->
+
+**Additional context**
+<!-- Add any other context or screenshots about the feature request here. -->
+
+## A/C
+- Make sure everything still works.
```

Comment:
I think we should add a commented-out set of acceptance criteria that the user can uncomment, e.g.
```
<!-- - Changes are covered by e2e tests -->
<!-- - Changes are covered by unit tests -->
<!-- - Documentation is created for any new or modified behavior. -->

Reply:
The requestor of the feature likely has not knowledge what e2e or unit tests the repo currently uses, they just know what they'd like the PoC to demonstrate.

Whether it is even practical to have specific tests is then upon the engineers reviewing the feature return and turning it into (in classical model) to Epics or similar.

Reply:
Yeah that definitely makes sense.

Reply:
I'd say let's just remove the acceptance criteria section

Reply:
> I'd say let's just remove the acceptance criteria section

@adelton wdyt?

Reply:
I agree.